install/kubernetes/ansible/istio/tasks/add_to_path.yml

```diff
@@ -1,3 +1,7 @@
+- name: Set istio dir
+  set_fact:
+    istio_dir: "{{ istio_k8s_dir | dirname | dirname }}"
```

Comment:
Shouldn't this be `{{ istio_k8s_dir | dirname }}` or is there a specific need for the second `dirname` ?

Reply:
@geoand  yes. `istio_k8s_dir` is  `/home/lilly/istio/install/kubernetes` and ISTIO_BIN_DIR is under `/home/lilly/istio/bin`, so it needs to get the parent of the parent of `istio_k8s_dir`.